compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/Utils.java

```diff
@@ -262,10 +286,11 @@ public static Schema getSchemaObject(Node node, SemanticModel semanticModel, Pro
             return null;
         }
 
-        InterfaceFinder interfaceFinder = new InterfaceFinder();
+        InterfaceEntityFinder interfaceFinder = new InterfaceEntityFinder();
         interfaceFinder.populateInterfaces(semanticModel);
+        // isSubgraph field is now set to false as we don't support subgraph schema generation yet
         SchemaGenerator schemaGenerator = new SchemaGenerator(serviceNode, interfaceFinder, semanticModel,
-                project, description);
+                project, description, false); // is subgraph is not set to false need to fix this
```

Comment:
I did not understand these comments 😄

Reply:
Seems I have missed to remove the second comment 😄. I have temporally added false flag here and this function used by the LS extension to get the Schema object.

Reply:
Now the code is updated to obtain correct subgraph flag